test/testsets.json

```diff
@@ -106,7 +106,7 @@
     { "name": "test_timers_error.js" },
     { "name": "test_timers_simple.js", "timeout": 10 },
     { "name": "test_tls_connect_and_close.js" },
-    { "name": "test_tty.js" },
+    { "name": "test_tty.js", "skip": ["all"], "reason": "need to setup test environment" },
```

Comment:
Skip the tty because the Python TestRunner doesn't monitor the terminal, will fix it later.